windows-02.md

```diff
@@ -1,26 +1,22 @@
-
 # Setting up for Node
-
 We don't do this until later in the semester, when we start using Node-based tools.
 
 ## Node.js
-
 Node is a Javascript runtime environment we will use to build news applications. This is where things get kinda dicey with my (lack of) Windows experience. Microsoft recommends using nvm-windows to install node, so let's go with that.
 
 - Follow [these directions to install nvm-windows](https://docs.microsoft.com/en-us/windows/nodejs/setup-on-windows) **BUT READ THE NEXT STEPS FIRST**.
   - When they say Launch Powershell, you should use **Git Bash** instead.
   - When it gets to installing Node.js **DON'T DO `nvm install latest`**.
-  - Instead follow the steps to do `nvm list available` and pick a the LTS version.
-  - Be sure to run the `nvm use <version>` they suggest from the LTS list.
-- In Git Bash, do `node --version` to make sure it worked. (v10.19.0 was the current stable version when this was written.)
+  - Instead, use this command: `nvm install v16.18.0`
+- In Git Bash, use the command `node --version` to make sure it worked.
+  - You should get a response that says you are using `v16.18.0`.
```

Comment:
I'm standardizing the node version between this and https://github.com/utdata/icj-project-rig.